source/component/tipue/PasDoc_Tipue.pas

```diff
@@ -244,12 +253,12 @@ procedure TipueAddFiles(Units: TPasUnits;
   end;
 
 const
-  TipueSearchCss: {$I tipuesearch.css.inc};
```

Comment:
Do not change the code formatting, esp. on lines where you didn't change anything :) In PasDoc we declare constants with types like

```pascal
const
  Name: Type = Value;
```

with no space between `Name` and `:`, and 1 space between `:` and `Type`.

The `Type = Value` in this case is defined in the include file.

The original code, as far as I see, was following our formatting. Please restore it.

Reply:
Sorry it's my habit, i'm not take care